frontend/libs/studio-feedback-form/src/FeedbackForm/Question/YesNoQuestion.tsx

```diff
@@ -26,7 +26,7 @@ export function YesNoQuestion({ id, label, value, buttonLabels, onChange }: YesN
 
   return (
     <div>
-      <StudioParagraph>{label}</StudioParagraph>
+      <StudioLabelAsParagraph>{label}</StudioLabelAsParagraph>
```

Comment:
Maybe we should add small size here too?🤔

Reply:
The font size for this question type matches the font size for the other question type (which is a textarea), so I think the size is ok
<img width="570" alt="Screenshot 2024-11-28 at 18 09 05" src="https://github.com/user-attachments/assets/0013cbdd-1b14-4a67-b9e4-85e0c94ba252">